src/main/java/io/camunda/zeebe/bpmnassert/testengine/ZeebeEngineImpl.java

```diff
@@ -16,7 +17,7 @@
 
 public class ZeebeEngineImpl implements ZeebeEngine {
 
-  public static final int PORT = 26500;
+  public static final int PORT = SocketUtil.getNextAddress().getPort();
```

Comment:
:x: this cannot be static if we want to have different ports for different instances